WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -1474,8 +1473,8 @@ extension AztecPostViewController : Aztec.FormatBarDelegate {
                                                    selectedRowIndex index: Int?,
                                                    onSelect: OptionsTableViewController.OnSelectHandler?) {
         // Hide the input view if we're already showing these options
-        if let optionsViewController = optionsViewController, optionsViewController.options == options {
-            if presentedViewController != nil {
+        if let optionsViewController = optionsViewController ?? (presentedViewController as? OptionsTableViewController), optionsViewController.options == options {
```

Comment:
So let's assume `self.optionsViewController == nil` then if `presentedViewController` is an `OptionsTableViewController` instance, then the `optionsViewController` constant in that if block won't be nil. However, the next line of code checks to make sure `self.optionsViewController != nil` which makes the conditional `??` sort of extraneous, right?